lambda-layer/otel-instrument

```diff
@@ -27,6 +27,10 @@ export OTEL_LOGS_EXPORTER=${OTEL_LOGS_EXPORTER:-"none"}
 # Enable Application Signals by default if not specified
 export OTEL_AWS_APPLICATION_SIGNALS_ENABLED=${OTEL_AWS_APPLICATION_SIGNALS_ENABLED:-"true"}
 
+if [ -z "${OTEL_METRICS_ADD_APPLICATION_SIGNALS_DIMENSIONS}" ]; then
+  export OTEL_METRICS_ADD_APPLICATION_SIGNALS_DIMENSIONS="true"
```

Comment:
Is this needed since the code is expected to default to `true`?

Also wondering, at first this looks like just a system property key:
```
OTEL_METRICS_ADD_APPLICATION_SIGNALS_DIMENSIONS =
      "otel.metrics.add.application.signals.dimensions";
```
Just curious, does `props.getBoolean(OTEL_METRICS_ADD_APPLICATION_SIGNALS_DIMENSIONS, true);` account for both System Property and Environment Variable?